### ISSUE TITLE
New Entry for Argentina

### ISSUE THREAD

**Issue by eogitbot** [w3c/wai-policies-prototype#368]
New Entry for Argentina

Submitted by [Fernando Petrone](mailto:fapetrone@gmail.com):

~~~yaml
---
country:
  en: Argentina
  # Manual enter other country names: 
updated: 2019-05-20
updatemsg:
# Related page:
province: 
policies:
  - title:
      en: Accesibilidad de la Información en las Páginas Web
    url: http://servicios.infoleg.gob.ar/infolegInternet/anexos/175000-179999/175694/norma.htm
    updated: 2010
    wcagver: WCAG 2.0, 
    enactdate: 2010
    type: law, Procurement law, 
    ministries:
      - title:
          es: "Oficina Nacional de Tecnologías de Información"
        url:
          es: "https://www.argentina.gob.ar/onti"
    webonly: true
    scope: 
    standard:
      - title:
          en: "WCAG 2.0"
        desc: "Conformance level A"
        url:
          en: "https://www.w3.org/TR/WCAG20/"
    documents:
      - title:
          es: "Web Accessibility Repository "
        desc: "The Web Accessibility Repository on GithHub contains many resources as guides, posters, technical recommendations, etc."
        url:
          es: "https://github.com/argob/accesibilidad-web"
---
~~~

**Comment by AndrewArch** [comment]
With some help from a Spanish speaking lawyer/a11y colleague, it is recommended to add [Regulation 2/2014](http://servicios.infoleg.gob.ar/infolegInternet/anexos/230000-234999/233667/norma.htm) be added as a relevant document - just checking that with Fernando who made the submission

**Comment by iadawn** [comment]
Addressed in #496
